yt_dlp/extractor/cbc.py

```diff
@@ -335,6 +371,7 @@ def _real_extract(self, url):
 
         formats = self._extract_m3u8_formats(m3u8_url, video_id, m3u8_id='hls')
         self._remove_duplicate_formats(formats)
+        self._find_secret_formats(formats, video_id)
```

Comment:
```suggestion
        formats.extend(self._find_secret_formats(formats, video_id))
```
(see the other suggestions)